sdk_container/src/third_party/coreos-overlay/coreos/config/env/sys-block/open-iscsi

```diff
@@ -0,0 +1,18 @@
+cros_post_src_install_tmpfiles() {
+	keepdir /var/lib/iscsi/ifaces
+
+	dosym ../flatcar/etc/iscsi/iscsid.conf /usr/share/iscsi/iscsid.conf
+	insinto /usr/share/flatcar/etc/iscsi
+	newins - iscsid.conf
```

Comment:
Is this a canonical way to install an empty file? TIL.

Reply:
The `-` is used to install a file from stdin. I just didn't pass anything to stdin here.

Reply:
Right. I've read that in the Gentoo PMS. I remember that when I wanted to install an empty file, I was doing `touch "${T}/empty-file"` and then do `newins "${T}/empty-file /whatever/path`. But this looks simpler. The only wrinkle is "In this case, it is an error if standard input is a terminal." - I guess it doesn't happen here.

Reply:
That didn't happen, even when I ran `emerge` locally. I think Portage just prevents it.